contributing.md

```diff
@@ -1,30 +1,77 @@
+
+- highly recommend by other contributors or fellow students
+- hard to find for various reasons
+- helpful if you struggle with a concept by a different way of teaching 
+- you wish you had known this resource when you started taking this course
+
+Since the selection process is community-driven, you'll get GitHub reactions from others for your submission. This gives you way to find out if you're on the right track.
+
+## Contributing
+
+Please [submit a pull request](#how-to-submit-a-pull-request) or [create an issue](#how-to-create-an-issue) to add something to the list.  
+**Submissions via pull requests are highly recommended. It may be daunting if you're new to Git and/or Github. But the use of this tools are important and useful skills for any developer. Need help getting started ? [Click here](#getting-started-with-git-and-github)**
+
+### Quality Standards for Issues and PRs
+
+- Make sure it hasn't already been submitted or declined
+- Submission must be in English
+- Check for proper spelling and grammar
+- Briefly write a description and why it should be added to the list
+
+## How to create an Issue
+
+- Proceed to the "Issues" tab in the top menu
+- Use the comment field for your submission
+- Apply the label `Submission` to issue
+
+## How to submit a Pull Request
 
-## Git Pull Request Style
+1. Create a personal fork of the project on Github via the button on the top right.
+
+2. Clone the fork on your local machine via the terminal command:
+```
+git clone  https://github.com/[your_username]/awesome-google-mobile-web-challenge-links-2017
+```
+
+3. Edit your changes locally.
+
+4. Push your branch to your fork on Github. First add all modified files:
+```
+git add .
+```
+
+5. Then update it:
+```
+git push master
+```
+
+6. Back to GitHub.com: From your fork open a pull request.
+
+---
+
+#### Getting started with Git and Github
+
+- Udacity Course: [How to Use Git and GitHub](https://www.udacity.com/course/how-to-use-git-and-github--ud775)
+- Udacity Course:  [Writing READMEs - Udacity course](https://www.udacity.com/course/writing-readmes--ud777)
+- Fun and Interactive GitHub Challenge: [Got 15 minutes and want to learn Git?](https://try.github.io/levels/1/challenges/1)
+- [Git - The simple guide](http://rogerdudler.github.io/git-guide/)
+
+#### Further Reading
 
 - [How to write the perfect pull request](https://github.com/blog/1943-how-to-write-the-perfect-pull-request)
+- [How to Write a Git Commit Message - The seven rules for a great commit message](http://chris.beams.io/posts/git-commit/#seven-rules)
+- [Distributed Git - Contributing to a Project]( http://www.git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#Commit-Guidelines)
+- [Thoughtbot - 5 Useful Tips For A Better Commit Message](https://robots.thoughtbot.com/5-useful-tips-for-a-better-commit-message)
+- [Tim Pope - A Note About Git Commit Messages](http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html)
```

Comment:
I would add that last but not least - if you are contributing (and you are not on contributor's list) please add yourself.